perf/Manifest.toml

```diff
@@ -245,8 +245,10 @@ uuid = "4ade58fe-a8da-486c-bd89-46df092ec0c7"
 version = "0.1.0"
 
 [[deps.ClimaLSM]]
-deps = ["ArtifactWrappers", "ClimaComms", "ClimaCore", "ClimaCoreTempestRemap", "Dates", "DiffEqCallbacks", "DocStringExtensions", "IntervalSets", "JLD2", "LinearAlgebra", "NCDatasets", "StaticArrays", "SurfaceFluxes", "Thermodynamics", "UnPack"]
-git-tree-sha1 = "9156a90b20e61afbbfb0c0ab0b11f2d6bb6c7ff1"
+deps = ["ArtifactWrappers", "CFTime", "ClimaComms", "ClimaCore", "ClimaCoreTempestRemap", "Dates", "DiffEqCallbacks", "DocStringExtensions", "IntervalSets", "JLD2", "LinearAlgebra", "NCDatasets", "StaticArrays", "SurfaceFluxes", "Thermodynamics", "UnPack"]
+git-tree-sha1 = "d9563c0ce883d71dd9558e12a7379a7ff84a52aa"
+repo-rev = "main"
+repo-url = "https://github.com/CliMA/ClimaLSM.jl.git"
```

Comment:
Same as `Manifest.toml` in `modular` folder - tracking a released version vs `main`.